standard/expressions.md

```diff
@@ -3688,6 +3819,55 @@ At run-time, the expression `await t` is evaluated as follows:
 
 An awaiter’s implementation of the interface methods `INotifyCompletion.OnCompleted` and `ICriticalNotifyCompletion.UnsafeOnCompleted` should cause the delegate `r` to be invoked at most once. Otherwise, the behavior of the enclosing async function is undefined.
 
+## §range-operator Range operator
+
+This operator provides a succinct syntax for specifying a (possibly empty) element range suitable for use in denoting a slice of an indexable sequence (§indexable-sequence).
+
+```ANTLR
+range_expression
+    : unary_expression
+    | range_expression? '..' range_expression?
+    ;
+```
+
+For an operation of the form `s .. e`, binary operator overload resolution ([§12.4.5](expressions.md#1245-binary-operator-overload-resolution)) is applied to select a specific operator implementation. The operands are converted to the parameter types of the selected operator, and the type of the result is the return type of the operator. *range_expression* shall have type `System.Index` or a type that can be converted implicitly to that type. Only one predefined range operator exists:
+
+> <!-- NotAn$Example: {} -->
+```csharp
+System.Range operator ..(System.Index start = 0, System.Index end = ^0);
+```
+
+The left and right operands denote, respectively, a start and end Index. For this operator, an object of (the immutable struct) type `System.Range` is returned that contains those Indexes. If the left operand is omitted, an Index of `0` is used. If the right operand is omitted, an Index of `^0` is used. As such,
+
+- `s .. e` is transformed by the implementation to `new System.Range(s, e)`.
+- `s ..` is transformed by the implementation to `new System.Range(s, ^0)`.
+- `.. e` is transformed by the implementation to `new System.Range(0, e)`.
+- `..` is transformed by the implementation to `new System.Range(0, ^0)`.
```

Comment:
“transformed by the implementation” is wrong here as:
- there is no requirement that the implementation does this transformation (and Roslyn does not always do so); and
- the Standard uses phrasing like “has the same meaning as” or “is equivalent to” (e.g. §12.8.8, §13.9.5)